packages/components/forms/src/base-input/types.ts

```diff
@@ -72,4 +72,8 @@ export interface BaseInputInternalProps extends CommonProps {
    * Allows to listen to an event when an element get focused
    */
   onFocus?: FocusEventHandler<HTMLTextAreaElement | HTMLInputElement>;
+  /**
+   * Allows setting size small to the input
+   */
+  size?: 'small' | 'default';
```

Comment:
Shouldn't we have the sizes as `small`, `medium` and so on as we have in other components? And have the medium being the default?

Reply:
Yes!
It's in the code style guide already
https://github.com/contentful/forma-36/blob/forma-v4/docs/code-style-guide.md#commonly-used-props

Reply:
@gui-santos I'll do this change then